lib/api/controllers/serverController.js

```diff
@@ -215,11 +215,12 @@ class ServerController extends BaseController {
 
           serviceName = current;
 
-          if (this.kuzzle.services.list[current].getInfos) {
-            return this.kuzzle.services.list[current].getInfos();
+          // @todo remove the condition when garbage collector is removed
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)